Dockerfile

```diff
@@ -72,15 +72,14 @@ ENV FSLDIR=/usr/share/fsl/5.0 \
     AFNI_IMSAVE_WARNINGS=NO \
     AFNI_TTATLAS_DATASET=/usr/share/afni/atlases \
     AFNI_PLUGINPATH=/usr/lib/afni/plugins \
-    PATH=/usr/lib/fsl/5.0:/usr/lib/afni/bin:$PATH
-
-# Installing and setting up ANTs
-RUN mkdir -p /opt/ants && \
-    curl -sSL "https://github.com/stnava/ANTs/releases/download/v2.1.0/Linux_Ubuntu14.04.tar.bz2" \
-    | tar -xjC /opt/ants --strip-components 1
-
-ENV ANTSPATH /opt/ants
-ENV PATH $ANTSPATH:$PATH
+ENV PATH=/usr/lib/fsl/5.0:/usr/lib/afni/bin:$PATH
+
+# Installing ANTs 2.2.0 (NeuroDocker build)
+ENV ANTSPATH=/usr/lib/ants
```

Comment:
Sticking with modified `ANTSPATH` to be consistent with NeuroDebian. Let me know if you'd rather keep `/opt/ants`. It's no big deal to me, either way.